src/app/certificate/page.tsx

```diff
@@ -0,0 +1,59 @@
+// 'use client';
+import { authOptions } from '@/lib/auth';
+import { getServerSession } from 'next-auth';
+import { CertificateComponent } from '@/components/Certificate';
+import db from '@/db';
+
+const CertificatePage = async () => {
+  const session = await getServerSession(authOptions);
+  const purchases = await db.userPurchases.findMany({
```

Comment:
This needs to change unfortunately since not all purchases are in this db
Some are fetched from appx
Let me take care of this